images/access_setup/content/bin/setup_kcp.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The pipelines-service Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$0")" >/dev/null
+  pwd
+)"
+
+usage() {
+  printf "Usage:
+    %s [options]
+
+Bootstrap a new KCP instance so it can be managed by pipeline as code
+
+Optional arguments:" "${0##*/}" >&2
+usage_args
+printf "
+Example:
+    %s -d -k /path/to/kcp.kubeconfig
+"  "${0##*/}" >&2
+}
+
+bootstrap_kcp() {
+  kcp_name="$(kubectl config view --minify -o json |
+    jq ".clusters[0].cluster.server" |
+    sed -r -e "s@.*://([^.]*).*@\1@").$(
+    kubectl kcp workspace current | cut -d\" -f2 | cut -d: -f2- | tr ':' '.'
```

Comment:
The script is making assumptions that are neither documented in usage nor in the README.
From what I see:
- KUBECONFIG environment variable is pointing to a kubeconfig providing access to kcp
- this kubeconfig has current context set to the kcp cluster and a user with the necessary rights
- the current context if for the right organisation workspace
I would rather see these things as parameter/environment variables with sane defaults. It would also help with documenting the behaviour and what can be expected

Reply:
I've tried to improve the documentation, and allowed the params to be passed by env vars. I do not want to deal with setting the context right now. That's an improvement that can be done later on.